packaging/bee.yaml

```diff
@@ -60,8 +60,10 @@ password-file: /var/lib/bee/password
 # resolver-options: []
 ## enable swap (default true)
 # swap-enable: true
-## swap ethereum blockchain endpoint (default "")
+## swap ethereum blockchain endpoint (default "") [deprecated]
```

Comment:
I would also remove "ethereum". The same is in other places.

Reply:
Done